src/main/java/ru/my/Mocito/departmentService/Idepartment.java

```diff
@@ -0,0 +1,27 @@
+package ru.my.Mocito.departmentService;
+
+import ru.my.Mocito.Employee;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public interface Idepartment {
+
+
+    Map<Integer, List<Employee>> depaertments();
+
+    ArrayList<Employee> employees(int department);
```

Comment:
```suggestion
    List<Employee> employees(int department);
```
надо завязываться на абстракции, а не реализации